spec/mixins/log_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+require 'routemaster/mixins/log'
+
+describe Routemaster::Mixins::Log do
```

Comment:
Nope. In the first implementation I changed the logic that was happening inside `_log` method then later we decided that for something that's there just to add syntactic sugar it had become too complex and we extracted big part of its own class and cleaned the content of this file. I think it's good to keep the file still in case we add something.